Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -63,32 +63,33 @@ script:
         return Object.keys(obj).length === 0;
     }
 
-    var mapTypeToValue = function (fields) {
-        var result = {};
-        var TYPE = 'type';
+    var fillgroupObject = function (fields, groupObject) {
+        var TYPE  = 'type';
         var VALUE = 'value';
+        var GROUP = 'group';
         if(fields && Array.isArray(fields)) {
             fields.forEach(function (element) {
-                if (element[TYPE] && element[VALUE]) {
-                    result[element[TYPE]] = element[VALUE];
+                if (element[TYPE] && element[VALUE] && element[GROUP]) {
```

Comment:
I thought about it, but there are too many cases we have to cover(If first have a group and second don't have a group, if there are elements with same group). we have to decide if we trust group field or covering all use cases